fusermount ss provided by package "fuse", but pyfuse3 requires package "fuse3",
which provides command fusermount3, not fusermount.

--- a/src/borg/helpers/fs.py
+++ b/src/borg/helpers/fs.py
@@ -335,7 +335,7 @@
 def umount(mountpoint):
     env = prepare_subprocess_env(system=True)
     try:
-        return subprocess.call(['fusermount', '-u', mountpoint], env=env)
+        return subprocess.call(['fusermount3', '-u', mountpoint], env=env)
     except FileNotFoundError:
         return subprocess.call(['umount', mountpoint], env=env)
 
